libcxx/include/variant

```diff
@@ -1273,6 +1293,24 @@ public:
     __impl_.__swap(__that.__impl_);
   }
 
+#  if _LIBCPP_STD_VER >= 26 && defined(_LIBCPP_HAS_EXPLICIT_THIS_PARAMETER)
+  struct __variant_visit_barrier_tag { // unnamable when using standard library modules
```

Comment:
I'm not sure what you try to say with the comment. I think it means you use an ugly name to avoid the std modules to fail. Is that correct? If this comment is not to useful. All ugly names are not exported.

Maybe instead add a comment like
```
// Helper class to implement [variant.visit]/10
//   Constraints: The call to visit does not use an explicit template-argument-list
//   that begins with a type template-argument.
```